components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -571,6 +571,23 @@ def testGetImage(self, gatewaywrapper, author_testimg_tiny):
         assert image.getOwnerOmeName == testImage.getOwnerOmeName
         assert image.getThumbVersion() is not None
 
+    def testGetImageLoadPixels(self, gatewaywrapper, author_testimg_tiny):
+        testImage = author_testimg_tiny
+        conn = gatewaywrapper.gateway
+        # By default, don't load pixels
+        image = conn.getObject("Image", testImage.id)
+        assert not image._obj.pixelsLoaded
+        # Load just the pixels
+        image = conn.getObject("Image", testImage.id,
+                               opts={'load_pixels': True})
```

Comment:
How much "value checking" do we need? We know the query is valid as it's loading the pixels. I'm only testing the new functionality - that the ```load_pixels``` option is being respected. If we're getting an invalid ```sizeX``` etc then something is very wrong (not just my query)! I could also check that ```pixelsType``` is also loaded if that's enough?